acceptance/tests/service_enable_start.rb

```diff
@@ -0,0 +1,12 @@
+test_name 'Validate that pxp-agent service can be enabled and started'
+
+agents.each do |agent|
+  step 'start service'
+  on(agent, puppet('resource service pxp-agent ensure=running enable=true'))
+
+  step 'validate that service is enabled and running'
+  on(agent, puppet('resource service pxp-agent')) do |result|
```

Comment:
Lines 7-12 are perhaps redundant, as during line 5 Puppet will check that the property values it tried to set were set.

The only thing we get from repeating the check explicitly is perhaps that there is a slight delay before we go back and check again.

I'd say keep the test as-is; because even if the asserts are technically redundant, readers of this code will see what it is that we're interested in testing. If the entire test was just line 5 it might be confusing to understand how its even a test!